src/Cassandra/Policies/Policies.cs

```diff
@@ -31,7 +33,7 @@ public static ILoadBalancingPolicy DefaultLoadBalancingPolicy
         {
             get
             {
-                return new TokenAwarePolicy(new DCAwareRoundRobinPolicy());
+                return new PartitionAwarePolicy();
```

Comment:
Let's create `PartitionAwarePolicy` by passing the `DCAwareRoundRobinPolicy()` explicitly here like how it was with `TokenAwarePolicy()`.